servant/servant.cabal

```diff
@@ -53,6 +52,7 @@ library
     , bytestring == 0.10.*
     , bytestring-conversion == 0.3.*
     , case-insensitive >= 1.2
+    , http-api-data
```

Comment:
Upper bound here as well (in fact this is the more important one, since other packages depend on this one).